src/app/proposals/shared/types.ts

```diff
@@ -0,0 +1,23 @@
+import Big from '@/lib/big'
+import { DecodedData } from '@/app/proposals/shared/utils'
+export interface IProposal {
```

Comment:
We're not using "I" in the codebase to show an interface. Could you rename to what this actually is?
To me it looks like a plain Proposal